sdks/python/Makefile

```diff
@@ -30,7 +30,7 @@ else
 		sed 's/io.k8s.apimachinery.pkg.apis.meta.v1.//' \
 		> $(WD)/swagger.json
 	cp ../../LICENSE $(WD)/LICENSE
-	$(DOCKER) openapitools/openapi-generator-cli:v5.4.0 \
+	$(DOCKER) openapitools/openapi-generator-cli:v7.4.0 \
```

Comment:
I think this was pinned to this version for a reason. Have you tested the generated SDK?

Reply:
Havent tested it yet but the sdk unit test is failing so I think the interface has changed. Do you remember the reason for the specific version pinning? Or is there some specific testing that would need to be done?